third_party/gtest-1.8.0/src/gtest.cc

```diff
@@ -3212,6 +3212,8 @@ void PrettyUnitTestResultPrinter::OnTestIterationEnd(const UnitTest& unit_test,
     PrintFailedTests(unit_test);
     printf("\n%2d FAILED %s\n", num_failures,
                         num_failures == 1 ? "TEST" : "TESTS");
+  } else {
+	  ColoredPrintf(COLOR_GREEN,  "[  ALL TESTS PASSED  ] \n");
```

Comment:
Have we considered pushing this change back to GoogleTest?  To me having an extra print saying that ALL TESTS PASSED makes sense.  Or should we be checking the return value of RUN_ALL_TESTS in our code to print this so they we have one less modification?